src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ScreenRecorder.cs

```diff
@@ -278,7 +278,12 @@ await CreateAndUploadGameplaySession(
         private async Task MoveSegmentsToProject(string botSegmentsDirectoryPrefix)
         {
             // get all the file paths normalized to /
-            var segmentFiles = Directory.EnumerateFiles(botSegmentsDirectoryPrefix).Where(a=>a.EndsWith(".json")).Select(a=>a.Replace('\\','/')).Select(a=>a.Substring(a.LastIndexOf('/')+1));
+            var segmentFiles = Directory.EnumerateFiles(botSegmentsDirectoryPrefix)
+                .Where(a=>a.EndsWith(".json"))
```

Comment:
Use a call to `BotSegmentDirectoryParser.OrderJsonFiles` instead of this new code block